src/core/prompts/system-prompt/components/tool_use/tools.ts

```diff
@@ -18,13 +18,19 @@ export async function getToolUseToolsSection(variant: PromptVariant, context: Sy
 	// (TODO tool is now dynamically added when focusChainEnabled is true)
 	const shouldIncludeTaskProgress = focusChainEnabled
 
+	// Define multi-root hint based on feature flag
+	const multiRootHint = context.isMultiRootEnabled
+		? ". In multi-workspace mode, you can specify a workspace using @workspace:path syntax (e.g., @frontend:src/index.ts)"
```

Comment:
We can remove`In multi-workspace mode` since they are already in this mode when we show the hint
```suggestion
		? "Use @workspace:path syntax (e.g., @frontend:src/index.ts) to specify a workspace."
```